package.json

```diff
@@ -8,9 +8,10 @@
     "start": "npm run build && node ./lib/server/index.js",
     "start-debug": "cross-env DEBUG=stackoverflowtags:* npm run start",
     "lint": "eslint ./test ./src/server/index.js",
-    "test": "npm run lint && mocha ./test/test.js",
+    "test": "npm run lint && mocha ./test/api.js",
```

Comment:
Better to have a separate `mocha` script & use `npm run lint && npm run mocha && npm run testcafe` here